QUEST.tsv

```diff
@@ -10,286 +10,286 @@ xQUEST_20150317_000009	Please select a starting zone.
+xQUEST_20150317_000018	Instead, I wandered through many far-seeing dreams to find an answer. {nl}You were at the end of them… a Savior.
+xQUEST_20150317_000019	However, I knew the demons would never rest while I kept my power. {nl}I was forced to hide away, until the time of your appearance.
+xQUEST_20150317_000020	I resolved to divide my being and subsist inside many of these messages. {nl}I continued to secretly create them, even as the kingdoms flew by over the millennia.
+xQUEST_20150317_000021	Savior, these messages are scattered across axes of time. {nl}Once they're all brought together, I will regain my complete form and help against the demons.
+xQUEST_20150317_000022	This divine task may be a heavy burden, but it is something you must bear. {nl}Perhaps, in the muddy midst of your trials and tribulations, something may come about.
 xQUEST_20150317_000023	Nevertheless, only you and I can save this world.
 xQUEST_20150317_000024	The first Paladin will hold the next one. {nl}I expect you at the highest flower garden.
-xQUEST_20150317_000025	The dark aura is indeed unpleasant… {nl}Yet, if this message is truly from the Goddess Laima, then I am simply amazed.
+xQUEST_20150317_000025	The dark aura is indeed unpleasant… {nl}But if this message is truly from the Goddess Laima, then I am at awe.
 xQUEST_20150317_000026	Well, you should hurry and go find Sir Uska. {nl}I'm sure he can tell you where to go next. {nl}May the Goddess bless your fate…
 xQUEST_20150317_000027	Please select the preferred Smart Gen.
 xQUEST_20150317_000028	William
-xQUEST_20150317_000029	The fee's 100 Silver. {nl}It'll be cheaper than cutting through monster central.
+xQUEST_20150317_000029	The fee's 100 silver. {nl}It'll be cheaper than cutting through monster central.
 xQUEST_20150317_000030	Cyneburg
-xQUEST_20150317_000031	I charge 100 Silver. {nl}That will cover operations, labor, and what have you.
+xQUEST_20150317_000031	I charge 100 silver. {nl}That will cover operations, labor and what have you.
 xQUEST_20150317_000032	Hey, wait a minute. {nl}You know you need more than that…
 xQUEST_20150317_000033	That's no good! {nl}I can't work for nothing, even for a Revelator.
 xQUEST_20150317_000034	Wandering Merchant
-xQUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such a curious way, perhaps you'll spare a moment to view my curiosities?
+xQUEST_20150317_000035	Like a dandelion's seeds, I am merely one who follows the wind. {nl}Since we've met in such a curious way, perhaps you could spare a moment to view my curiosities?
 xQUEST_20150317_000036	Kristina
-xQUEST_20150317_000037	Welcome! {nl}You're looking for a travel companion, right? {nl}You won't find a better place for partners than here!
+xQUEST_20150317_000037	Welcome! {nl}You're looking for a travel companion, right? {nl}Then you've come to the right place!
```

Comment:
Big change in meaning without the word "partner". I would go with something like "This is the right place to find a partner!"